private/dependency_tree_parser.bzl

```
@@ -293,7 +293,34 @@ def _generate_imports(repository_ctx, dep_tree, explicit_artifacts, neverlink_ar
                 target_import_string.append("\tvisibility = [%s]," % (",".join(["\"%s\"" % v for v in default_visibilities])))
                 alias_visibility = "\tvisibility = [%s],\n" % (",".join(["\"%s\"" % v for v in default_visibilities]))
 
-            # 9. Finish the java_import rule.
+            # 9. If `strict_visibility` is True in the artifact spec, define public
+            #    visibility only for non-transitive dependencies.
```

This needs an update, probably a left-out copy from the above code.
```suggestion
            # 9. If `license_info ` is passed, load license info for the artifact.
```